README.md

```diff
@@ -0,0 +1,20 @@
+<img alt="Netlify Status"
+     src="https://api.netlify.com/api/v1/badges/269f7467-6afd-49ae-97f2-61a160e93a9a/deploy-status">
```

Comment:
This url is broken (404), @domenkozar can you get a working one pls?

Reply:
let's just remove it, using cloudflare pages now that don't have a badge

Reply:
Removed.